timely/src/progress/reachability.rs

```diff
@@ -74,7 +74,6 @@
 
 use std::collections::{BinaryHeap, HashMap, VecDeque};
 use std::cmp::Reverse;
-
```

Comment:
Hrm. Not sure about the formatting, but generally try to space out `std` vs crate stuffs!